ci-operator/config/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master.yaml

```diff
@@ -130,6 +130,14 @@ tests:
   optional: true
   steps:
     cluster_profile: azure4
+    test:
```

Comment:
Does this override the default tests? We want to ensure we continue running the default tests in addition to the new one.

Reply:
I wouldn't think this would override anything. It's marked optional and is meant to be called by a periodic, right?

Reply:
Looking at the latest run, I don't see openshift-tests being executed (yet). Perhaps they would be after the additional tests? I want to ensure they both execute before moving forward with this change.
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/42126/rehearse-42126-pull-ci-openshift-cloud-credential-operator-master-e2e-azure-manual-oidc/1689504388621537280/artifacts/e2e-azure-manual-oidc/

Reply:
It appears this does override the default test. I think we need to add an entry to `tests:`, preferably above the e2e-test-azident

`- ref: openshift-e2e-test`